azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/sqlserver/AlwaysEncryptedAutoConfiguration.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.microsoft.azure.spring.autoconfigure.sqlserver;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.config.BeanPostProcessor;
+import org.springframework.boot.autoconfigure.AutoConfigureBefore;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceProperties;
+import org.springframework.boot.autoconfigure.jdbc.JndiDataSourceAutoConfiguration;
+import org.springframework.boot.autoconfigure.jdbc.XADataSourceAutoConfiguration;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+import org.springframework.jdbc.datasource.embedded.EmbeddedDatabaseType;
+
+import javax.sql.DataSource;
+
+@Configuration
+@ConditionalOnClass({DataSource.class, EmbeddedDatabaseType.class})
+@EnableConfigurationProperties({AlwaysEncryptedDataSourceProperties.class, KeyVaultProperties.class})
+@ConditionalOnProperty(name = AEConstants.PROPERTY_AE_ENABLED)
+@AutoConfigureBefore({DataSourceAutoConfiguration.class, JndiDataSourceAutoConfiguration.class,
+        XADataSourceAutoConfiguration.class})
+public class AlwaysEncryptedAutoConfiguration {
+    private static final Logger LOG = LoggerFactory.getLogger(AlwaysEncryptedAutoConfiguration.class);
+
+    /**
+     *
+     * @return post processor bean that initializes KeyVault ofr SQL Driver
```

Comment:
Suggest to remove this comment since:
1.  `KeyVaultProviderInitializer` is not a post processor
2.  `initializes key vault` doesn't carry extra information compared to `KeyVaultProviderInitializer`
3.  typo of `ofr SQL Driver`